test/errors/unsupportedAsIdent2-detailed.good

```diff
@@ -0,0 +1,8 @@
+─── error in unsupportedAsIdent2.chpl:6 [UnsupportedAsIdent] ───
+  This form of 'as' is not yet supported.
+  'as' requires both the original and new names to be simple identifiers, but instead got the following:
```

Comment:
Can we explicitly say whether or not it's the original or the new name that's incorrect? I suspect we can by comparing against the `As` node's children. It would make this error a bit less complicated (`'as' requires the original name to be a simple identifier, but in the following code, it isn't:` e.g.)

Reply:
Yes. We also know whether it's the original or new name in the code that reports this error, so it could even be a boolean flag in the error info. But I'll try using the more robust approach of checking against the children first.